k8s/openebs-operator.yaml

```diff
@@ -743,7 +747,7 @@ spec:
         - name: OPENEBS_IO_INSTALLER_TYPE
           value: "openebs-operator"
         - name: OPENEBS_IO_HELPER_IMAGE
-          value: "quay.io/openebs/linux-utils:1.4.0"
+          value: "openebs/linux-utils:1.12.0"
```

Comment:
Can you also update this with `ci` images, so we can test the changes wit multi-arch image. @akhilerm

Reply:
Done.